shared/templates/audit_rules_usergroup_modification/oval.template

```diff
@@ -18,7 +18,7 @@
   </ind:textfilecontent54_test>
   <ind:textfilecontent54_object id="object_audit_rules_usergroup_modification_{{{ NAME }}}_augen" version="1">
     <ind:filepath operation="pattern match">^/etc/audit/rules\.d/.*\.rules$</ind:filepath>
-    <ind:pattern operation="pattern match">^\-w[\s]+{{{ PATH }}}[\s]+\-p[\s]+\b([rx]*w[rx]*a[rx]*|[rx]*a[rx]*w[rx]*)\b[\s]+(-k[\s]+|-F[\s]+key=)\w+[\s]*$</ind:pattern>
+    <ind:pattern operation="pattern match">^\-w\s+{{{ PATH }}}\s+\-p\s+(wa|aw)\s+(-k[\s]+|-F[\s]+key=)[-\w]+\s*$</ind:pattern>
```

Comment:
What is the reason of this regex update? I think the original intention was to accept more variations of permissions, e.g. when someone wants to have also read permission alongside the permissions required by the `audit_rules_usergroup_modification` rules it was accepted by the original OVAL check, now it would not be and the OVAL check would be asserted as fail in such cases.

Reply:
Actually my intention was to make sure the both permissions related to write were assessed. Before the update, if the rule was only with a or w, it would be wrongly assessed as PASS. But, reading your comment now I realized that combinations of wa with other permissions were excluded with this update. I will fix it to solve both cases. Thanks